cl/opinion_page/views.py

```diff
@@ -264,6 +264,24 @@ def view_docket(request: HttpRequest, pk: int, slug: str) -> HttpResponse:
     sort_order_asc = True
 
     de_list = docket.docket_entries.all().prefetch_related("recap_documents")
+    de_as_list = list(de_list)
+    first_entry = de_as_list[:1]
+    n = min(len(de_as_list) - 1, 4)
+    if n < 4:
+        last_entries = de_as_list[-n:]
+    else:
+        last_entries = de_as_list[-n:]
+        current = last_entries[0]
+        n = n - 1
+        while (
+            n > 0
+            and current.date_filed
+            and de_as_list[n].date_filed
+            and de_as_list[n].date_filed == current.date_filed
+        ):
+            current = de_as_list[n]
+            n = n - 1
+            last_entries.insert(0, current)
```

Comment:
I think I'd approach this a bit differently, with two goals in mind. **First,** unfortunately, some dockets have tens of thousands of entries, so pulling all of those into a list is...not great for the poor database or for our application tier's memory. It's important we never do that (we've made that mistake before). 

**Second**, we shouldn't show the quick look if we already have those same documents on the page. A lot of dockets are really short (if they're new, say), so it's silly to show a quick look when you can already see the whole thing. It's also unnecessary to show the initial complaint when you're on page one of a DESC docket. It'd be half an inch away already. 

So with that in mind, I think what we need here is probably three queries:

 - The current one to get the usual stuff, and then paginate it. If there's more than one page, then:
     - If the initial complaint isn't in the selected page (i.e., it's not page 1):
         - A second query to show the initial complaint. This should probably just query `entry_number=1`, I think? 
     - If we're not on the last page:
         - A third query for the last few items.
 - Else, if there's only one page, no quick look needed; just do the one query.

I think if we don't have the initial complaint, it makes sense to show an error in the quick look saying as much. If we don't have the last few items, we won't know it. 

That sound OK?

Reply:
I disagree regarding "we shouldn't show if..."  The quick look is there for those that want to open it and look.  If they can see everything they want, they wont open it.

If they don't see what they want, they will open it and it will be in a standard view.

The goal is to not have to scroll to see what you are interested in.

Reply:
Having looked at a lot of dockets, scrolling isn't usually the issue so much as pagination. I don't want to repeat content in the quick look that's already there on the page.

Reply:
I disagree with your opinion.

I'll code up the changes per your suggestions and push up the commits shortly.